users/test/test_views.py

```diff
@@ -1,21 +1,134 @@
+import json
+from typing import Any
+
 from django.contrib.auth import get_user_model
 from django.contrib.auth.tokens import PasswordResetTokenGenerator
 from django.utils.encoding import smart_bytes
 from django.utils.http import urlsafe_base64_encode
 from faker import Faker
 from rest_framework import status
+from rest_framework.exceptions import PermissionDenied
 from rest_framework.reverse import reverse
-from rest_framework.test import APIRequestFactory, APITestCase
+from rest_framework.test import APIClient, APIRequestFactory, APITestCase
 
-from users.views import PasswordResetAPIView, PasswordResetEmailView
+from users.views import (
+    PasswordResetAPIView,
+    PasswordResetEmailView,
+    UserFollowView,
+)
 
 fake = Faker()
 User = get_user_model()
+follow_view = UserFollowView.as_view()
 password_reset_email_view = PasswordResetEmailView.as_view()
 password_reset_api_view = PasswordResetAPIView.as_view()
 
 
+class TestFollowingView(APITestCase):
+    def setUp(self) -> None:
+        self.password = fake.password()
+        self.user_one = User.objects.create_user(
+            username=fake.name(), email=fake.email(), password=self.password
+        )
+        self.user_two = User.objects.create_user(
+            username=fake.name(), email=fake.email(), password=self.password
+        )
+        self.client = APIClient()
+        self.factory = APIRequestFactory()
+
+    @property
+    def bearer_token(self) -> Any:
+        login_url = reverse("login")
+        response = self.client.post(
+            login_url,
+            data={"email": self.user_two.email, "password": self.password},
+            format="json",
+        )
+        token = json.loads(response.content).get("access")  # type: ignore
```

Comment:
don't do a generic ignore ignore a specific error code

Reply:
It was the only solution offered from a django-stubs error. Mypy doesn't recognize the data or content of the response object.